ui/src/components/Rules/index.tsx

```diff
@@ -120,17 +120,23 @@ const Rules: React.FC = () => {
             />
           </div>
         </div>
-
-        <div>
+        <h1 className="mb-3 text-lg font-bold text-zinc-200">
+          {'Rules Listing'}
+        </h1>
+        <ul className="xs:collection-cards-vertical">
           {(data as IRuleGroup[]).map((el) => (
-            <RuleGroup
-              onDelete={refreshData}
-              onEdit={editHandler}
+            <li
               key={el.id}
-              group={el as IRuleGroup}
-            />
+              className="collection relative mb-5 flex h-fit transform-gpu flex-col overflow-hidden rounded-xl bg-zinc-800 bg-cover bg-center p-4 text-zinc-400 shadow ring-1 ring-zinc-700 xs:w-full sm:mb-0 sm:mr-5"
```

Comment:
The collections page cards have slightly more padding inside, this will align the two:

```suggestion
              className="collection relative mb-5 flex h-fit transform-gpu flex-col overflow-hidden rounded-xl bg-zinc-800 bg-cover bg-center p-7 text-zinc-400 shadow ring-1 ring-zinc-700 xs:w-full sm:mb-0 sm:mr-5"
```

Reply:
Replaced with the same CSS from the collections page. This change made the rule card a little too "squished" in my opinion.